staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -376,19 +374,6 @@ func cleanDryRun(u *url.URL) string {
 	return strings.Join(utilsets.NewString(dryRun...).List(), ",")
 }
 
-func cleanUserAgent(ua string) string {
```

Comment:
Consider instead making this choose between e.g. {"Browser", "kubectl", "self"}?

I'm happy to approve it either way but this avoids a change, has known cardinality, and remains at least a little useful.

Reply:
After our offline discussion, do you mind if I mark this as resolved?